themes/gatsby-theme-blog-core/gatsby-node.js

```diff
@@ -209,3 +130,59 @@ exports.onCreateNode = ({ node, actions, getNode, createNodeId }) => {
     createParentChildLink({ parent: fileNode, child: node })
   }
 }
+
+// These templates are simply data-fetching wrappers that import components
+const PostTemplate = require.resolve(`./src/templates/post`)
+const PostsTemplate = require.resolve(`./src/templates/posts`)
+
```

Comment:
Perhaps we should we rename these to something more descriptive? E.g. post-query.js I find it odd currently when searching through files as both post.js come up and I have to check to distinguish them.

Reply:
yeah, I thought about this for sure when I was translating it to the -core theme. `posts` and `post` can change in -core, but it would be a breaking change for shadowing in `gatsby-theme-blog`.